dotnet/src/SemanticKernel.Skills/Skills.Memory.Qdrant/Skills.Memory.Qdrant.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+    </PropertyGroup>
+    <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+    <PropertyGroup>
+        <AssemblyName>Microsoft.SemanticKernel.Skills.Memory.VectorDB</AssemblyName>
```

Comment:
As Shawn noted, update this namespace.

Reply:
Okay copying local version which has this change into commit.